src/assisted_test_infra/download_logs/download_logs.py

```diff
@@ -329,16 +329,27 @@ def get_ui_url_from_api_url(api_url: str):
 
 @JunitTestCase()
 def download_must_gather(kubeconfig: str, dest_dir: str):
-    log.info(f"Downloading must-gather to {dest_dir}, kubeconfig {kubeconfig}")
+    must_gather_dir = f"{dest_dir}/must-gather-dir"
+    os.mkdir(must_gather_dir)
+
+    log.info(f"Downloading must-gather to {must_gather_dir}, kubeconfig {kubeconfig}")
     command = (
         f"oc --insecure-skip-tls-verify --kubeconfig={kubeconfig} adm must-gather"
-        f" --dest-dir {dest_dir} > {dest_dir}/must-gather.log"
+        f" --dest-dir {must_gather_dir} > {must_gather_dir}/must-gather.log"
     )
     try:
         run_command(command, shell=True, raise_errors=True)
+
     except RuntimeError as ex:
         log.warning(f"Failed to run must gather: {ex}")
 
+    log.debug("Archiving %s...", must_gather_dir)
+    with tarfile.open(f"{dest_dir}/must-gather.tar", "w:gz") as tar:
```

Comment:
Did you make sure the tar file produced here is exactly the same as the one in regular OCP jobs?

1. Exactly same extension
2. Exactly same compression
3. Exactly same directory structure inside the tar

Reply:
yep, same thing as other jobs
there's one exception, but I think it's not significant which is regular OCP jobs also have an empty dir named ``.`` inside of the tar:
![Screenshot from 2022-03-07 11-17-59](https://user-images.githubusercontent.com/7492909/157002634-42f558d4-e802-4658-ab11-ef0ccd9629ea.png)